addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -17,13 +17,13 @@
                             <i t-elif="!user.im_status" class="fa fa-fw fa-question-circle" title="No IM status available"/>
                         </span>
                     </span>
-                    <div class="d-flex flex-column o_card_user_infos">
+                    <div class="d-flex flex-column o_card_user_infos overflow-hidden">
                         <span class="fw-bold" t-esc="user.name"/>
                         <t t-if="userInfoTemplate" t-call="{{userInfoTemplate}}"/>
-                        <a t-if="email" t-att-href="'mailto:'+email">
+                        <a t-if="email" t-att-href="'mailto:'+email" class="text-truncate">
                             <i class="fa fa-fw fa-envelope me-1"/><t t-esc="email"/>
                         </a>
-                        <a t-if="phone" t-att-href="'tel:'+phone">
+                        <a t-if="phone" t-att-href="'tel:'+phone" class="text-truncate">
                             <i class="fa fa-fw fa-phone me-1"/><t t-esc="phone"/>
                         </a>
```

Comment:
- `user.name` could also be long, so `text-truncate` could be worth being added too.
- In general when text could be truncated, it's worth having some means to see the whole content, e.g. with `title` attribute

Reply:
Hello @alexkuhn, I get your point but I just want to double check. IMO chances that the name are long enough to wrap are very low and on top of that it looks like the most important information here, so we should be very careful with it and allow it to wrap rather than hiding some letters.

Reply:
@chgo-odoo I got some friends from Africa that have very long names, so I can see some cases where this could be a problem. However I get this is a very small minority of potential users of Odoo that could be affected by this issue.

Reply:
Thanks for the context, indeed we want this to be as much accessible as possible, I've made the change !